os/booting-on-vmware.md

```diff
@@ -68,7 +64,12 @@ Run VMware Workstation GUI:
 3. Name your VM, then click `Import`
 4. (Press `Retry` *if* VMware Workstation raises an "OVF specification" warning)
 5. Edit VM settings if necessary
-6. Start your CoreOS VM
+6. Create a cloud-config image (see below) containing at least at least one valid SSH key using the instructions at:
+    https://github.com/coreos/coreos-cloudinit/blob/master/Documentation/config-drive.md
+    * On Fusion, simply connect and mount this image as the CD, ignoring the instruction below regarding the filesystem label which are not applicable.
```

Comment:
Maybe make it clear that you are attaching the config drive? I think that repeating that phrase here will work well:

...simply connect and mount this config-drive image as the CD